src/validatePrTitle.js

```diff
@@ -2,24 +2,37 @@ const conventionalCommitsConfig = require('conventional-changelog-conventionalco
 const conventionalCommitTypes = require('conventional-commit-types');
 const parser = require('conventional-commits-parser').sync;
 
-module.exports = async function validatePrTitle(prTitle) {
+const defaultTypes = Object.keys(conventionalCommitTypes.types);
+
+module.exports = async function validatePrTitle(prTitle, types = defaultTypes) {
   const {parserOpts} = await conventionalCommitsConfig();
   const result = parser(prTitle, parserOpts);
 
+  function printAvailableTypes() {
```

Comment:
Sneaked in another improvement here: https://github.com/amannn/action-semantic-pull-request/issues/45